chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/BrokenSurveillanceRulesEmailJob.java

```diff
@@ -97,14 +97,12 @@ public void execute(JobExecutionContext jobContext) throws JobExecutionException
         String subject = null;
         String htmlMessage = null;
         if (jobContext.getMergedJobDataMap().getString("type").equalsIgnoreCase("All")) {
-            String subjectSuffix = env.getProperty("oversightEmailAcbWeeklySubjectSuffix");
-            subject = getAcbNamesAsCommaSeparatedList(jobContext) + " " + subjectSuffix;
-            htmlMessage = env.getProperty("oversightEmailAcbWeeklyHtmlMessage");
+            subject = env.getProperty("oversightEmailAcbWeeklySubjectSuffix");
+            htmlMessage = env.getProperty("oversightEmailAcbWeeklyHtmlMessage") + " " + getAcbNamesAsCommaSeparatedList(jobContext);
```

Comment:
It should be more like String.format(env.getProperty("oversightEmailAcbWeeklyHtmlMessage"), getAcbNamesAsCommandSeparatedList(jobContext)); Most of our more recent jobs use that style of creating a message although I think this one is older so that's why it isn't written that way already.